src/tr1d1um/communication_test.go

```diff
@@ -162,99 +171,55 @@ func TestHandleResponse(t *testing.T) {
 	})
 
 	t.Run("GoodRDKResponse", func(t *testing.T) {
-		fakeResponse := newTestingHTTPResponse(http.StatusOK, "") //these arguments are irrelevant as we mock RDK response below
-		extractedData := []byte(`{"statusCode": 202}`)
+		RDKResponse := []byte(`{"statusCode": 202}`)
+		wrpMsg := wrp.Message{
+			Type:    wrp.SimpleRequestResponseMessageType,
+			Payload: RDKResponse}
+
+		var encodedData []byte
+		errEncoding := wrp.NewEncoderBytes(&encodedData, wrp.Msgpack).Encode(wrpMsg)
```

Comment:
You can use `wrp.MustEncode` for this.  It simplifies testing a bit, as it will simply panic in the event of an error, which will abort the test.